test/spec/exceptions/application_validator.spec.js

```diff
@@ -319,10 +319,10 @@ describe('ApplicationValidator', () => {
         });
       });
       context('with unknown options', () => {
-        it('should fail', () => {
+        it('should not throw an error', () => {
```

Comment:
Can you check here that the logger's debug function is called?

Reply:
@MathieuAA : Got it; done. Sorry I wasn't sure how to do this previously; my knowledge on Chai is limited; but now I learnt a lot :smile:

Reply:
And let me know if you see any further issues.

Reply:
Ah! I should have mentioned where a similar example could be found. Sorry!!